core/build_playerglobal/src/lib.rs

```diff
@@ -273,7 +276,10 @@ fn rust_method_name_and_path(
     // generate a reference to the function pointer that should exist
     // at that path in Rust code.
     let path_tokens = TokenStream::from_str(&path).unwrap();
-    quote! { Some((#flash_method_path, #path_tokens)) }
+    (
+        quote! { Some((#flash_method_path, #path_tokens)) },
```

Comment:
I don't like this duplication, but I can't think of a better way to do it since the `flash_method_path` generation closely follows the `path` generation